resources/views/front/campaigns.blade.php

```diff
@@ -61,6 +63,9 @@
                     <div class="m-1">
                         {!! Form::select('featured_until', ['' => __('front.campaigns.open.filters.featured'), 0 => __('general.yes'), 1 => __('general.no')], request()->get('featured_until'), ['class' => 'form-control']) !!}
                     </div>
+                    <div class="m-1">
+                        {!! Form::select('genre', ['' => __('campaigns.fields.genre')] + $genres->getGenres(), request()->get('genre'), ['class' => 'form-control']) !!}
```

Comment:
getGenres() has a param for an empty option, looks like it could be merge with the __('campaigns.fields.genre')